src/ltn12.lua

```diff
@@ -22,9 +24,6 @@ _M.source = source
 _M.sink = sink
 _M.pump = pump
 
-local unpack = unpack or table.unpack
```

Comment:
If `unpack()` is being used anywhere in this file, this actually serves a purpose as a shim for different Lua version compatibility and should not be removed. If it isn't being used at all then of course it is cruft, but please confirm that because it isn't the same issue as the "duplicated" code removed in the rest of this commit.

Reply:
the exact same declaration is at line 12, so this is fine